shell/ci/release/docker.sh

```diff
@@ -173,7 +173,13 @@ build_and_push_image() {
   fi
   (
     set -x
-    docker buildx build "${args[@]}"
+
+    if [[ $OSTYPE == "linux-gnu"* ]]; then
+      docker buildx --builder devbase build "${args[@]}"
+    else
+      docker build "${args[@]}"
```

Comment:
Still need `buildx`, just not `--builder`.

```suggestion
      docker buildx build "${args[@]}"
```

Reply:
Ah I was trusting copilot too much this time